README.md

```diff
@@ -1,2 +1,51 @@
-# module-ballerinax-microsoft.outlook.calendar
-Connects to Microsoft Calendar using Ballerina.
+Ballerina Microsoft Outlook Connector
+======================================
+
+[![Build Status](https://github.com/ballerina-platform/module-ballerinax-microsoft.outlook.calendar/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-microsoft.outlook.calendar/actions?query=workflow%3ACI)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-microsoft.outlook.calendar.svg)](https://github.com/ballerina-platform/module-ballerinax-microsoft.outlook.calendar/commits/master)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+The `microsoft.outlook.calendar` is a [Ballerina](https://ballerina.io/) connector for Microsoft Outlook Calendar.
+
+[Microsoft Outlook Calendar](https://www.microsoft.com/en-ww/microsoft-365/outlook.calendar) is composed of the calendar and scheduling component of Outlook developed by Microsoft.
+
+This connector provides operations for connecting and interacting with Microsoft Graph API Outlook Calendar endpoints over the network. 
+For more information about configuration and operations, go to the module. 
+- [ballerinax/microsoft.outlook.calendar](https://docs.central.ballerina.io/ballerinax/microsoft.outlook.calendar/1.0.1)
+
+## Building from the source
+### Setting up the prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
+
+   * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
+
+   * [OpenJDK](https://adoptopenjdk.net/)
+
+        > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
+
+2. Download and install [Ballerina Beta2](https://ballerina.io/). 
+
+### Building the source
+Execute the commands below to build from the source after installing Ballerina SL Beta 2 version.
```

Comment:
We don't need to mention this right? Since it's a prerequisite step?

Reply:
Yeah. We will remove